argo_qa/schema.xml

```diff
@@ -11,55 +11,31 @@
-    <dynamicField name="*_tiv"   type="text" stored="true" indexed="true"  multiValued="false" termVectors="true" termPositions="true" termOffsets="true"/>
-    <dynamicField name="*_timv"  type="text" stored="true" indexed="true"  multiValued="true"  termVectors="true" termPositions="true" termOffsets="true"/>
-    <dynamicField name="*_tsiv"  type="text" stored="true" indexed="true"  multiValued="false" termVectors="true" termPositions="true" termOffsets="true"/>
-    <dynamicField name="*_tsimv" type="text" stored="true" indexed="true"  multiValued="true"  termVectors="true" termPositions="true" termOffsets="true"/>
-
-    <!-- English text (_te...) -->
-    <dynamicField name="*_tei"    type="text_en" stored="true" indexed="true"  multiValued="false"/>
-    <dynamicField name="*_teim"   type="text_en" stored="true" indexed="true"  multiValued="true"/>
-    <dynamicField name="*_tes"    type="text_en" stored="true" indexed="false" multiValued="false"/>
-    <dynamicField name="*_tesm"   type="text_en" stored="true" indexed="false" multiValued="true"/>
-    <dynamicField name="*_tesi"   type="text_en" stored="true" indexed="true"  multiValued="false"/>
-    <dynamicField name="*_tesim"  type="text_en" stored="true" indexed="true"  multiValued="true"/>
-    <dynamicField name="*_teiv"   type="text_en" stored="true" indexed="true"  multiValued="false" termVectors="true" termPositions="true" termOffsets="true"/>
-    <dynamicField name="*_teimv"  type="text_en" stored="true" indexed="true"  multiValued="true"  termVectors="true" termPositions="true" termOffsets="true"/>
-    <dynamicField name="*_tesiv"  type="text_en" stored="true" indexed="true"  multiValued="false" termVectors="true" termPositions="true" termOffsets="true"/>
-    <dynamicField name="*_tesimv" type="text_en" stored="true" indexed="true"  multiValued="true"  termVectors="true" termPositions="true" termOffsets="true"/>
-
     <!-- string (_s...) -->
-    <dynamicField name="*_si"    type="string"    stored="true" indexed="true"  multiValued="false"/>
-    <dynamicField name="*_sim"   type="string"    stored="true" indexed="true"  multiValued="true"/>
+    <dynamicField name="*_si"    type="string"    stored="false" indexed="true"  multiValued="false"/>
+    <dynamicField name="*_sim"   type="string"    stored="false" indexed="true"  multiValued="true"/>
     <dynamicField name="*_ss"    type="string"    stored="true" indexed="false" multiValued="false"/>
     <dynamicField name="*_ssm"   type="string"    stored="true" indexed="false" multiValued="true"/>
     <dynamicField name="*_ssi"   type="string"    stored="true" indexed="true"  multiValued="false"/>
     <dynamicField name="*_ssim"  type="string"    stored="true" indexed="true"  multiValued="true"/>
-    <dynamicField name="*_ssort" type="alphaSort" stored="true" indexed="true"  multiValued="false"/>
+    <dynamicField name="*_ssort" type="alphaSort" stored="false" indexed="true"  multiValued="false"/>
```

Comment:
we don't need to display sort fields, only sort on them -- they do not need to be retrievable.